internal/app/wait4x/cmd/rabbitmq.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 Mohammad Abdolirad
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"errors"
+	"github.com/atkrad/wait4x/pkg/checker/rabbitmq"
+	"github.com/atkrad/wait4x/pkg/waiter"
+	"github.com/spf13/cobra"
+)
+
+// NewRabbitMQCommand creates the rabbitmq sub-command
+func NewRabbitMQCommand() *cobra.Command {
+	rabbitmqCommand := &cobra.Command{
+		Use:   "rabbitmq DSN",
+		Short: "Check RabbitMQ connection.",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return errors.New("DSN is required argument for the rabbitmq sub-command")
+			}
+
+			return nil
+		},
+		Example: `
+  # Checking RabbitMQ connection
+  wait4x rabbitmq 'amqp://127.0.0.1:5672'
+
+  # Checking RabbitMQ connection with credentials
+  wait4x rabbitmq 'amqp://guest:guest@127.0.0.1:5672'
```

Comment:
Please also put a sample about `vhost` (if we support it).

Reply:
I'm not sure. Let me please to check it.

Reply:
Done